features/logging/logging_acceptance.feature

```diff
@@ -0,0 +1,163 @@
+@clusterlogging
+@commonlogging
+Feature: Logging smoke test case
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-37508
+  @admin
+  @destructive
+  Scenario: One logging acceptance case for all cluster
+# Deploy cluster-logging operator via web console
+    Given logging service is removed successfully	
+    Given the logging operators are redeployed after scenario	
+    Given logging channel name is stored in the :logging_channel clipboard	
+    Given I register clean-up steps:	
+    """	
+    Given logging service is removed successfully	
+    Then the step should succeed	
+    """	
+    Given "cluster-logging" packagemanifest's catalog source name is stored in the :clo_opsrc clipboard	
+    Given "elasticsearch-operator" packagemanifest's catalog source name is stored in the :eo_opsrc clipboard	
+
+    Given I switch to the first user	
+    Given the first user is cluster-admin	
+    Given I open admin console in a browser	
+    # subscribe cluster-logging-operator	
```

Comment:
I suggest to subscribe EO first, we have a bz that if we deploy CLO first, sometimes the elasticsearch CR elasticsearch couldn't be created, not sure if this issue has been fixed or not.

Reply:
The bz https://bugzilla.redhat.com/show_bug.cgi?id=1752073 had been fixed, please ignore this comment